src/Interpreters/JoinedTables.h

```diff
@@ -30,14 +30,13 @@ class JoinedTables
     }
 
     StoragePtr getLeftTableStorage();
-    bool resolveTables();
+    bool resolveTables(bool include_all_columns);
```

Comment:
Does this ever get called with different flag values for the same JoinedTables object? Maybe we could make it a const field of JoinedTables instead.